netdisco/discoverables/buderus.py

```diff
@@ -0,0 +1,13 @@
+"""Discover Buderus KM200 gateways."""
+from . import MDNSDiscoverable
+
+
+class Discoverable(MDNSDiscoverable):
+    """Add support for discovering Buderus KM200."""
+
+    def __init__(self, nd):
+        """Initialize the Buderus KM200 discovery."""
+        super(Discoverable, self).__init__(nd, '_http._tcp.local.')
+
+    def get_entries(self):
+        return self.find_by_device_name('iCom ')

```

Comment:
This seems not related to Buderus name ?

Reply:
The module is branded Buderus KM200 sold in Belgium and Germany, it seems also valid for Buderus KM100 and KM50. This is sold in others country under the brand Junkers, module MB LAN.
It is also sold under Bosch brand.
all brand above are owned by Bosch Company.

The application that uses this on phones is called Bosch Easy Remote. this application detects the module via same way as this pull request : mDNS with"_http._tcp.local." and name begin with "iCom". That's the way they do it, I checked in apk source ccode.

Reply:
Okay, so if it's used by more than just Buderus, it shouldn't be called Buderus.